recipe/msys2-pkgs.py

```diff
@@ -181,29 +206,58 @@ def get_depends(pkg):
     seen[pkg] = depends, spdx, desc, url, src_name
 
 
-meta = f"""package:
+meta = f"""recipe:
   name: m2-binary-packages
-  version: {date}
-
-source:"""
+  version: "{date}"
+"""
 
 sources_template = """
-  - url:
-      - https://repo.msys2.org/msys/{{ msys_type }}/{{ url_base }}
```

Comment:
Let's keep both URLs.

Reply:
I ran into an issue where the script was using one url and rattler-build was using the other and the sha hash didnt match. I switched the order now to hopefully avoid that.

Reply:
Hmm, that's weird. Do you remember which? The two should be identical.

Reply:
I think it was wget. Let me check.

Reply:
I couldnt reproduce it, perhaps something was wrong in my local cache. I reverted the change.